macros/seed_existance_check.sql

```diff
@@ -0,0 +1,69 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+{% macro seed_existance_check() %}
+  {{ return(adapter.dispatch('seed_existance_check', 'snowplow_unified')()) }}
+{% endmacro %}
+
+{% macro default__seed_existance_check() %}
+    {% if execute %}
+        {# Ensure that this check is only performed during 'run' or 'build' commands #}
+        {# Log the flags.WHICH#}
+        {%- if flags.WHICH in ('run', 'run-operation') -%}
+            {% for node in graph.nodes.values() | selectattr("resource_type", "equalto", "seed") | selectattr("package_name", "equalto", "snowplow_unified") %}
+
```

Comment:
Does this technically force them to have our seed files loaded? I was thinking the reason we have the variables is so they can use their own file/table if they don't want to use ours. It's not that big a deal, they can just seed our files then do that and this is a nice catch for anything we add in the future as well I guess?

Reply:
Should we focus on that now or keep it for the future?